### ISSUE TITLE
Multiomic trajectories (General Question)

### ISSUE THREAD

**Issue by dsalinasduron-msmary** [cole-trapnell-lab/monocle3#764]
Hi all,

I was wondering if Monocle 3 can be used to compute multiomic trajectories. I have found work on creating co-embeddings that have both chromatin accessibility (ATAC-seq) and RNA sequencing features, but I cannot find work on computing and interpreting trajectories on such embeddings. In principle, the principal-graph-based trajectories computed by Monocle 3 may be learned on multiomic co-embeddings. However, I am wondering if there has been any work (published or otherwise) on this by the Trapnell lab, and  if so, whether this was a fruitful investigation or considered an unproductive endeavor. (I apologize if this is not the right venue for this sort of question.)

Thank you for your time and for your work on Monocle 3.
